### DISCUSSION TITLE
Increase number of categories + add export/import list + filter and more

### DISCUSSION THREAD

**Discussion by QuentinHuber** [community/community#13116] [Other Feature Feedback, Questions, & Ideas]
### Increase number of categories & add export/import folder categories + filter

I find this option super convenient.
Categorizing repositories can help us improve our monitoring.
I'm a bit of a jack-of-all-trades person, so I quickly made a folder for the web repositories, another for the "awesome", the projects with raspberries... it's so long that I quickly reached the maximum, i.e. 16.

Could you increase the number to 25 (or more 😉) it's more than enough for a good organization without too much info, or add subclasses, for example in my web folder, I could divide it into (Front/Back/CleanCode/Docker/Test...).

// MORE OPTION //

- Add a filter to know which ones are without a list.

- Redirection when returning to the previous page after modifying the name of a list. (recurring error 404 for Alt + ← or ⌘ + Z followers) or when searching via uri.
_And simply see us again on the basic page, that is to say: https://github.com/QuentinHuber?tab=stars_

- Create a page list because _https://github.com/stars/QuentinHuber/lists_ returns an error 404 or redirection to _https://github.com/QuentinHuber?tab=stars_ 

- See whether to keep the page https://github.com/stars/QuentinHuber which is an interface without list

- Add an option of an export/import for example in a sheet file (.xsl .csv .ods .html or .tsv) of a table which would include the name of the category with the links of this repository.

Exemple : 

|A                |B                                                        |C|
|------------|----------------------------------------|--|
|❤️ Github |https://github.com/github/feedback|
|                  |https://github.com/github/gitignore|
|                  |https://github.com/github/fetch|
|                  |https://github.com/github/opensource.guide|
|                  |https://github.com/github/copilot-docs|

### **...**

Thanks for the myriad of open source projects, and to see you trying to improve the user experience of github. Bisous 😘

**Comment by zach-is-my-name** [comment]
I also would like more categories than 16
